### ISSUE TITLE
Inconsistency between Myrkrobe results and assembly

### ISSUE THREAD

**Issue by Cashalow** [iqbal-lab/Mykrobe-predictor#146]
Hello,

I am using Mykrobe to detect resistance on reads from a staphylococcus aureus isolate.
Among the results, two mutations conferring resistance to quinolones are detected : 
gyrA S84L
grlA/parC S80F
with a high confidence: more than 200 alt_depth against 0 wt depth.

However, I assembled the genome an checked the corresponding proteins, and found that both mutations were absent. I aligned the reads on the assembled genome in order to see if I was not missing a mixed population (but the sequencing was performed directly on the strain) but I could not detect any mixed signal at those position. I aligned with bwa.

Have you already observed such problem ? 

Thanks for your help and for the cool software !

**Comment by iqbal-lab** [comment]
Sorry for the slow reply @Cashalow . I don't think we have seen this before - @Phelimb ?

**Comment by Cashalow** [comment]
Thanks for your answer ! Do you want me to provide some fastq data associated with this problem so you can check you can replicate my problem ?

**Comment by iqbal-lab** [comment]
Probably a good idea. I'm afraid I'm about to go on vacation for a week. Will get back to you ASAP after.
In the meantime can you give the precise version of mykrobe, what operating system and the output json please

**Comment by Cashalow** [comment]
I'm using the version on bioconda which is mykrobe v0.5.6-0-gbd7923a-dirty, on Linux 
Can I send you the rest of the data through your professional email ?

**Comment by iqbal-lab** [comment]
Yes but about yo get on plane for vacation. Back in 10 days. Apologies for delay

**Comment by iqbal-lab** [comment]
I've realised this is still open @sachalau - many apologies. I think I/we probably need to send the data. My email is zi AT ebi.ac.uk - send me a mail and we'll figure out how to get it to us.

**Comment by sachalau** [comment]
I sent you the data a while ago from my professional mail I think. Thanks for getting back to me.

**Comment by iqbal-lab** [comment]
Argh. Any hint as to your email address or email title?
